Makefile-libostree.am

```diff
@@ -149,6 +148,7 @@ endif
 
 if USE_LIBSOUP
 libostree_1_la_SOURCES += \
+    $(ENUM_GENERATED) \
```

Comment:
The problem is a bit messier than this...since other things may use generated enums.

And also, we don't want to generate enums for private headers in the first place, which is another bug.  I had a half-working patch to fix this...let me see if I can dig that up.

Are you actually planning to use `--with-soup=no`?